base/test/valveModule_tests.cpp

```diff
@@ -401,4 +395,158 @@ BOOST_AUTO_TEST_CASE(valve_relay)
     BOOST_TEST(sink1Que->size() == 2);
     BOOST_TEST(sink2Que->size() == 2);
 }
+
+BOOST_AUTO_TEST_CASE(multiType_frames)
```

Comment:
since there are no comments it is not very clear as to what the test is trying to acheive. a suggestion here is to make use of optional message argument to BOOST_TEST(condition, message) to express what is being tested. 
_Note_ I am **not** asking you to add code comments but still improve the readability by using the abive and better variable namies may be.

Reply:
additionally there is BOOST_TEST_MESSAGE(..) which can be used to add more expressiveness.

Reply:
Testcases are refactored using BOOST_TEST(condition, message) and variable names are updated.